sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/ListingFileCatalog.scala

```diff
@@ -64,11 +66,18 @@ class ListingFileCatalog(
   }
 
   override def refresh(): Unit = {
+    refresh0(true)
```

Comment:
nit: instead of having a parameter in `refresh0`, can we just call `refresh0` here, and then invalidate the file status cache?